gameswap-service/gameswap.yml

```diff
@@ -64,12 +65,12 @@ logging:
     - type: console
     - type: file
       # The file to which current statements will be logged.
-      currentLogFilename: /var/log/gameswap/gameswap.log
+      currentLogFilename: /tmp/log/gameswap/gameswap.log
```

Comment:
Oh yeah, since i'm on OSX, didn't know if that was going to be an issue for you.